### PR TITLE
Milestone 3: Feature Contact Form

### PR DESCRIPTION
**Pull request by taldr27**
## Section Contact Form
In this milestone we: 
- Added the contact form to the page.
- Added background and some style configuration on the page.
- Added a rule to be able to apply fonts in the styles.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -163,20 +169,41 @@ <h3>Skills</h3>
...
 
-
-
-
+        <section class="contact">
+            <p class="name">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            <div class="div-contact">
+                <form class="contact-form" action="https://formspree.io/f/xayknejl" method="post">
+                    <input type="text" maxlength="30" placeholder="Full Name" required>
+                    <input type="email" placeholder="Email address" required>
```
Comment (mgmediaweb) [inline index.html]: Kindly, the name and email fields must contain the NAME attribute for the form submission to work correctly.

File: index.html [hunk 2]
```diff
@@ -163,20 +169,41 @@ <h3>Skills</h3>
...
+                    <textarea placeholder="Enter text here" rows="7"></textarea>
```
Comment (mgmediaweb) [inline index.html]: Kindly, the message field should have a limit of 500 characters, I suggest you use max-length for that.

Comment (mgmediaweb) [reply index.html]: Kindly, the message field must be required to send the message, I suggest you use REQUIRED for that.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @taldr27 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost4

## Highlights
- Professional README file. ✔️
- PR has a good title and summary. ✔️
- Good HTML Structure. ✔️

## Required Changes ♻️

Kindly, the font used in your project is not the same as the original Figma design, I suggest you search for it in Google Fonts and apply it to your project.

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi Team👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality ✅
📌 Good commit messages ✅
📌Professional README file. ✅
📌 Great design 🔥🔥

## Optional suggestions
- [ OPTIONAL ] I recommend add the information for the another author so kindly add it to your `README` 💯🚀🔥 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**